test/index.spec.js

```diff
@@ -228,25 +229,193 @@ describe('FlipMove', () => {
     });
   });
 
-  describe('duration prop runtime checking', () => {
-    let wrapper;
-    beforeEach(() => {
-      wrapper = shallow(<FlipMove />);
+  describe('prop runtime checking and conversion', () => {
+    let errorStub;
+    let warnStub;
+    const SFC = () => null;
```

Comment:
Huh! I remember that SFCs couldn't return `null`, but apparently that was fixed a year and a half ago @_@, before React 15 @_@